modules/candidate_list/jsx/candidateListIndex.js

```diff
@@ -112,6 +122,13 @@ class CandidateListIndex extends Component {
           <a href={url}>{cell}</a></td>
       );
     }
+
+    if (column === 'Visit Label' || column === 'Subproject') {
```

Comment:
is subproject already getting send as array?

Reply:
see line 73 - I'm doing the same as for visit label